io/src/main/scala/sbt/io/IO.scala

```diff
@@ -309,13 +309,13 @@ object IO {
 
   /**
    * Creates a temporary directory and provides its location to the given function.  The directory
-   * is deleted after the function returns.
+   * is deleted after the function returns, unless `keepDirectory` is set to true.
    */
-  def withTemporaryDirectory[T](action: File => T): T =
+  def withTemporaryDirectory[T](action: File => T, keepDirectory: Boolean = false): T =
```

Comment:
Plz add override instead of default param - http://www.scala-sbt.org/0.13/docs/Coding-Guideline.html#Prefer+method+overloading+over+default+parameter+values